itm/itm_scenario_runner.py

```diff
@@ -5,6 +5,8 @@
 from abc import ABC, abstractmethod
 import os
 
+def get_swagger_class_enum_values(klass):
+    return [getattr(klass,i) for i in dir(klass) if not i.startswith("_") and isinstance(getattr(klass,i), str)]
```

Comment:
I had hoped this would be easier, but was surprised at how many options that I thought would work did not.  If there is a better way to filter, let me know.  This is one of the reasons that I'm unsure if this is worth it.  The reduction in duplicate code is a big deal, but is this function reliable?

Reply:
My quick look at the changes make me think this is great and will help a lot-- I expect there to be a lot more fields with controlled vocabs, so this sets a template/precedent for how to implement them in the future.  There could be numerous places in TA3 and TA2 ADM code that will want the list of valid values.

Why do you ask if it's reliable?  Do you mean it sometimes doesn't work or gives unexpected results?

Reply:
The function makes some assumptions that _should_ always be true, especially since we are auto generating the code.  I would have preferred a function that returns all the class member variables and then filter out the known swagger extras.